pkgs/development/python-modules/ircstates/default.nix

```diff
@@ -4,24 +4,33 @@
 , pythonOlder
 , irctokens
 , pendulum
+, freezegun
+, cachetools
 , python
 }:
 
 buildPythonPackage rec {
   pname = "ircstates";
-  version = "0.11.8";
+  version = "0.12.0";
   disabled = pythonOlder "3.6";  # f-strings
 
   src = fetchFromGitHub {
     owner = "jesopo";
     repo = pname;
     rev = "v${version}";
-    sha256 = "0scxqcgby4vzh2q937r0wy2mb46aghjf47q3z6fp6di1b6hlj7zh";
+    sha256 = "0nrgahw9pw5y0bc48fib85zmn7g1k04qqlgsqbxqpdnvkn7f4mk1";
   };
 
+  postPatch = ''
+    sed -iE 's/freezegun.*/freezegun/' requirements.txt
+    sed -iE 's/cachetools.*/cachetools/' requirements.txt
+  '';
+
   propagatedBuildInputs = [
     irctokens
     pendulum
+    freezegun
```

Comment:
This is a tests only dependency and should go in `nativeCheckInputs`.

Reply:
```
ERROR: Could not find a version that satisfies the requirement freezegun (from ircstates) (from versions: none)
ERROR: No matching distribution found for freezegun
```

Should I just remove the freezegun line from requirements.txt entirely? I'm not particularly familiar with Python packaging nor the Nixpkgs builder.

Reply:
@jessopo Awoo.

Reply:
I'll fix after cccamp

Reply:
> I'll fix after cccamp

(a little nerdsniping later...)

Thanks @jesopo!